pyomo/contrib/mindtpy/tests/test_mindtpy.py

```diff
@@ -34,11 +38,12 @@ def test_OA_8PP(self):
             opt.solve(model, strategy='OA',
                       init_strategy='rNLP',
                       mip_solver=required_solvers[1],
-                      nlp_solver=required_solvers[0])
+                      nlp_solver=required_solvers[0],
+                      bound_tolerance=1E-5)
 
             # self.assertIs(results.solver.termination_condition,
             #               TerminationCondition.optimal)
-            self.assertTrue(fabs(value(model.cost.expr) - 68) <= 1E-2)
+            self.assertAlmostEqual(value(model.cost.expr), 68, places=1)
```

Comment:
Could this be replaced with `model.get_objective_value()`? It should be the same of course, I'm just curious.

Reply:
Pyomo has no function of `model.get_objective_value()`. The same answer as above.